tests/unit/eyes_common/test_json_utils.py

```diff
@@ -91,7 +91,8 @@ def test_running_session_serialization_and_deserialization():
     assert rs == json_utils.attr_from_json(rs_json, RunningSession)
 
 
-def test_multithread_seialization():
+@pytest.mark.parametrize("i", range(5))
+def test_multithreading_serialization(i):
```

Comment:
Is this unused "i" argument here to make test run 5 times?

Reply:
Exactly